qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/SnapshotReplicationTest.java

```diff
@@ -77,9 +77,16 @@ public void shouldReplicateSnapshots() {
   public void shouldReceiveLatestSnapshotOnRejoin() {
     // given
     final var leaderNodeId = clusteringRule.getLeaderForPartition(1).getNodeId();
-    final var otherBrokers = clusteringRule.getOtherBrokerObjects(leaderNodeId);
-    final var firstFollowerId = otherBrokers.get(0).getConfig().getCluster().getNodeId();
-    final var secondFollowerId = otherBrokers.get(1).getConfig().getCluster().getNodeId();
+    final var followers =
+        clusteringRule.getOtherBrokerObjects(leaderNodeId).stream()
+            .map(b -> b.getConfig().getCluster().getNodeId())
+            .collect(Collectors.toList());
+
+    // Temp fix to make sure we don't restart broker-0 due to
+    // https://github.com/zeebe-io/atomix/issues/208
+    final var firstFollowerId = followers.stream().filter(nodeId -> nodeId != 0).findFirst().get();
+    final var secondFollowerId =
+        followers.stream().filter(nodeId -> nodeId != firstFollowerId).findFirst().get();
```

Comment:
Nit: by using implicit variable inference with `var` above, the type of `firstFollowerId` is now Number (i.e. it was not implicitly unboxed by casting it to `int`), so `!=` checks object instance equality. I mean it works here but not sure if this is what you intended :sweat_smile: